platforms/Windows/android_sdk/android_sdk.wxs

```diff
@@ -0,0 +1,512 @@
+      </Component>
+      <Component Directory="AndroidSDK_usr_lib_swift_android_ARCH">
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\libswiftSwiftOnoneSupport.so" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="Synchronization" Directory="Synchronization.swiftmodule">
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\Synchronization.swiftmodule\$(ArchTriple).swiftdoc" />
+      </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\Synchronization.swiftmodule\$(ArchTriple).swiftinterface" />
+      </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\Synchronization.swiftmodule\$(ArchTriple).swiftmodule" />
+      </Component>
+      <Component Directory="AndroidSDK_usr_lib_swift_android_ARCH">
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\libswiftSynchronization.so" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="apinotes" Directory="AndroidSDK_usr_lib_swift_apinotes">
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\apinotes\std.apinotes" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="libcxxshim" Directory="AndroidSDK_usr_lib_swift_android">
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\libcxxshim.h" />
+      </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\libcxxshim.modulemap" />
+      </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\libcxxstdlibshim.h" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="Registrar" Directory="AndroidSDK_usr_lib_swift_android_ARCH">
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\lib\swift\android\$(ArchArchDir)\swiftrt.o" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="Configuration">
+      <Component Directory="AndroidSDK">
+        <File Source="$(SDK_ROOT)\SDKSettings.plist" />
+      </Component>
+      <Component Directory="AndroidPlatform">
+        <File Source="$(PLATFORM_ROOT)\Info.plist" />
+      </Component>
+    </ComponentGroup>
+
+    <!-- Features -->
+    <Feature Id="AndroidSDK" AllowAbsent="no" Title="!(loc.AndroidSdk_ProductName_$(ProductArchitecture))">
+      <ComponentGroupRef Id="XCTest" />
+      <ComponentGroupRef Id="SwiftRemoteMirror" />
+      <ComponentGroupRef Id="BlocksRuntime" />
+      <ComponentGroupRef Id="libdispatch" />
+      <ComponentGroupRef Id="_Concurrency" />
+      <ComponentGroupRef Id="_Differentiation" />
+      <ComponentGroupRef Id="_RegexParser" />
+      <ComponentGroupRef Id="_StringProcessing" />
+      <ComponentGroupRef Id="_Volatile" />
+      <ComponentGroupRef Id="Android" />
+      <ComponentGroupRef Id="Cxx" />
+      <!-- FIXME: <ComponentGroupRef Id="CxxStdlib" /> -->
```

Comment:
Is there a GH issue associated with this?

Reply:
filing one

Reply:
https://github.com/apple/swift/issues/73855